en/docs/integrations/build-your-first-integration.md

```diff
@@ -0,0 +1,59 @@
+# Build Your First Integration
+Choreo allows you to efficiently build integrations by connecting APIs, events, and streams. Choreo also provides a comprehensive set of prebuilt integration templates that you can select and use depending on your requirement.
+
+This quick start guide walks you through the steps to quickly configure and try out the **GitHub New Issue to Google Sheets Row** integration in Choreo.
+
+## Prerequisites:
```

Comment:
The repetitive words `integration configuration` is a bit odd and confusing to me. Can't we avoid that repetition and simply say:
```
Following are the prerequisites for the integration configuration.
- A GitHub account.
- A Google account.
- A blank spreadsheet named Sample GitHub Integration with a default worksheet named Sheet1 created in the above Google account.  That spreadsheet name and worksheet name will be used in the configuration.
```

Reply:
Updated the prerequisites section.